### ISSUE TITLE
Karma 0.13.9 doesn't work with IE<9

### ISSUE THREAD

**Issue by mgol** [karma-runner/karma#1564]
With Karma 0.13.9 IE<9 no longer get captured. I didn't have such problems with Karma 0.12.24.

To see the problems, first make sure you have `BROWSER_STACK_USERNAME` & `BROWSER_STACK_ACCESS_KEY` environmental variables set up properly to your BrowserStack account data. Then checkout [Sizzle](https://travis-ci.org/jquery/sizzle)'s commit https://github.com/jquery/sizzle/commit/c76829c and run:
```
npm install
npm test
```
You'll see that at least the `karma:desktop`, `karma:ios` & `karma:oldIe` targets work fine (you can kill the process afterwards, the rest doesn't matter here.

Then change Karma to 0.13.9 & grunt-karma to 0.8.3 (or checkout the [ie-doesnt-work](https://github.com/mzgol/sizzle/tree/ie-doesnt-work) branch from my fork where it's set up this way) and run:
```
npm install
npm test
```
again. `karma:desktop` &, `karma:ios` should pass again but you'll then see:
```
Running "karma:oldIe" (karma) task
17 08 2015 23:55:19.484:INFO [karma]: Karma v0.13.9 server started at http://localhost:9876/
17 08 2015 23:55:19.489:INFO [launcher]: Starting browser ie 6.0 (Windows XP) on BrowserStack
17 08 2015 23:55:19.490:INFO [launcher]: Starting browser ie 7.0 (Windows XP) on BrowserStack
17 08 2015 23:55:19.490:INFO [launcher]: Starting browser ie 8.0 (Windows 7) on BrowserStack
17 08 2015 23:55:25.506:INFO [launcher.browserstack]: ie 7.0 (Windows XP) session at https://www.browserstack.com/automate/builds/57633abc95c7d50511cefe33bf53e74df4876acd/sessions/d70ac260f01ccc86b980977a05ba22813327476f
17 08 2015 23:55:25.532:INFO [launcher.browserstack]: ie 6.0 (Windows XP) session at https://www.browserstack.com/automate/builds/57633abc95c7d50511cefe33bf53e74df4876acd/sessions/bc7eaf843242d0de1788f852a84684c1a6a12c3d
17 08 2015 23:55:25.716:INFO [launcher.browserstack]: ie 8.0 (Windows 7) session at https://www.browserstack.com/automate/builds/57633abc95c7d50511cefe33bf53e74df4876acd/sessions/643f6e191bde5edaeaf369dd566f1dd3250431b7
18 08 2015 00:00:30.112:WARN [launcher.browserstack]: ie 7.0 (Windows XP) has not captured in 300000 ms, killing.
18 08 2015 00:00:33.131:WARN [launcher.browserstack]: ie 6.0 (Windows XP) has not captured in 300000 ms, killing.
18 08 2015 00:00:33.505:INFO [launcher.browserstack]: ie 7.0 (Windows XP) session at https://www.browserstack.com/automate/builds/57633abc95c7d50511cefe33bf53e74df4876acd/sessions/1274e6da370de9848a9d1af9b95ac3e72468db58
18 08 2015 00:00:34.915:WARN [launcher.browserstack]: ie 8.0 (Windows 7) has not captured in 300000 ms, killing.
18 08 2015 00:00:36.211:INFO [launcher.browserstack]: ie 6.0 (Windows XP) session at https://www.browserstack.com/automate/builds/57633abc95c7d50511cefe33bf53e74df4876acd/sessions/04cf53654c029951bacf73dc68d7ca684912dd52
18 08 2015 00:00:37.998:INFO [launcher.browserstack]: ie 8.0 (Windows 7) session at https://www.browserstack.com/automate/builds/57633abc95c7d50511cefe33bf53e74df4876acd/sessions/d036746d059fd40ffb80e2d57a8855f94fc000f0
```
and so on - until everything times out and the process is killed.

EDIT: Added info about setting up BrowserStack environmental variables.

**Comment by Dignifiedquire** [comment]
Thanks for the detailed report, will look into it as soon as I find some time

**Comment by Dignifiedquire** [comment]
I've contacted browser stack for a proper account so I can setup tests on travis to ensure we do not miss this again, and I can test this properly and fix

**Comment by Dignifiedquire** [comment]
I've started working on this, but I'm not able to connect to IE6 for some reason. @mgol do you have any idea what could be going wrong?

**Comment by Dignifiedquire** [comment]
@JamesMGreene if you have any ideas for IE6 help I'd appreciate it as well. 

WIP PR is up at #1813

**Comment by andersonba** [comment]
```27 01 2016 17:12:04.955:WARN [launcher.browserstack]: ie 9.0 (Windows 7) has not captured in 90000 ms, killing.```

**Comment by dignifiedquire** [comment]
Released [`0.13.20`](https://github.com/karma-runner/karma/releases/tag/v0.13.20) which supports `forceJSONP` and now should work with IE >= 7
